15_perms_lab/permslab.txt

```diff
@@ -13,3 +13,20 @@
 3. Read the output of "help umask". Explain in your own words what umask does, and how would you use it to set new files permissions to ------r--
 
 
+
+1. cd wildcards
+2. chmod a=r main.c
+3. chmod 562 player.h
+4. Not sure what happend, but using the following chmod, it "kicked" me out from "Doc" dir.
+How can I revers it?
+5. chmod -R 600 README.txt text.c

```

Comment:
That command did not cause issues, but I suspect you tried some variant of it that did. For example running `chmod 600 Doc` would cause issues.

The reason is you lose execute permission on the directory itself, and without it can't do anything in it.

The command you wrote should work, as well as `chmod 600 Doc/*`